test/unit/controllers/declaration/SealControllerSpec.scala

```diff
@@ -53,18 +59,18 @@ class SealControllerSpec extends ControllerSpec with ScalaFutures with ErrorHand
 
       "display page method is invoked and cache is empty" in new SetUp {
 
-        val result = controller.displayPage(Mode.Normal)(getRequest())
-
-        status(result) must be(OK)
+//        val result = controller.displayPage(Mode.Normal)(getRequest())
```

Comment:
you can just put pending in first line of tests to disable them - they will still compile but not count to results